CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The Call Message Events feature in the Twilio Voice React Native SDK, previously
 
 - **(Breaking)** The error code for attempting to send a call message with a payload size exceeding maximum limits has changed from `31209` to `31212`.
 
+- **(Breaking)** The behavior of `call.sendMessage` has now changed. When sending a message such that the value of `content` is type `string`, the content will no longer be processed through `JSON.stringify` before being sent to Twilio. If the value of `content` is any type other than `string`, it will behave as original.
+
```

Comment:
There was no error beforehand. It was just that the strings would be wrapped with quotes when they probably should not have been.

Reply:
To clarify, there were no errors in the backend before this change, and there will be no errors after this change. The hypothetical that I presented was purely that, just in case something in the future occurs such that the backend expects a non-quote delimited string: `'foo' | 'bar'`. And if we had passed `'foo'`, it would be incorrectly delimited to `'"foo"'`. Hence why I did not think we should `JSON.stringify` at all. I still have my preference against doing it at all, but we already had that conversation.

Reply:
It seems we are not on the same page even after our last discussion. I don't like the solution either but we need to satisfy to following for a better customer experience. Do you have a better solution?

- If the contentType is application/json, the customer should be able to provide a JSON object directly
- If the contentType is not application/json, the customer should be able to provide an encoded string of that type